dbcs/honda_crv_2017_can.dbc

```diff
@@ -0,0 +1,301 @@
+
+BO_ 231 XXX_4: 5 ADAS
+
+BO_ 304 GAS_PEDAL2: 8 PCM
+ SG_ ENGINE_TORQUE_ESTIMATE : 7|16@0- (1,0) [-1000|1000] "Nm"  NEO
+ SG_ ENGINE_TORQUE_REQUEST : 23|16@0- (1,0) [-1000|1000] "Nm"  NEO
+ SG_ CAR_GAS : 39|8@0+ (1,0) [0|255] ""  NEO
+
+BO_ 330 STEERING_SENSORS: 8 EPS
+ SG_ STEER_ANGLE : 7|16@0- (-0.1,0) [-500|500] "deg"  NEO
+ SG_ STEER_ANGLE_RATE : 23|16@0- (-1,0) [-3000|3000] "deg/s"  NEO
+ SG_ STEER_ANGLE_OFFSET : 39|8@0- (-0.1,0) [-128|127] "deg"  NEO
+ SG_ STEER_WHEEL_ANGLE : 47|16@0- (-0.1,0) [-500|500] "deg"  NEO
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 344 POWERTRAIN_DATA: 8 PCM
+ SG_ XMISSION_SPEED : 7|16@0+ (0.002759506,0) [0|70] "m/s"  NEO
+ SG_ ENGINE_RPM : 23|16@0+ (1,0) [0|15000] "rpm"  NEO
+ SG_ XMISSION_SPEED2 : 39|16@0+ (0.002759506,0) [0|70] "m/s"  NEO
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 380 POWERTRAIN_DATA2: 8 PCM
+ SG_ PEDAL_GAS : 7|8@0+ (1,0) [0|255] ""  NEO
+ SG_ ENGINE_RPM : 23|16@0+ (1,0) [0|15000] "rpm"  NEO
+ SG_ GAS_PRESSED : 39|1@0+ (1,0) [0|1] ""  NEO
+ SG_ ACC_STATUS : 38|1@0+ (1,0) [0|1] "rpm"  NEO
+ SG_ BOH_17C : 37|5@0+ (1,0) [0|1] "rpm"  NEO
+ SG_ BRAKE_LIGHTS_ON : 32|1@0+ (1,0) [0|1] "rpm"  NEO
+ SG_ BOH2_17C : 47|10@0+ (1,0) [0|1] "rpm"  NEO
+ SG_ BRAKE_PRESSED : 53|1@0+ (1,0) [0|1] ""  NEO
+ SG_ BOH3_17C : 52|5@0+ (1,0) [0|1] "rpm"  NEO
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 401 GEARBOX: 8 PCM
+ SG_ GEAR_SHIFTER : 5|6@0+ (1,0) [0|63] ""  NEO
+ SG_ GEAR : 35|4@0+ (1,0) [0|15] ""  NEO
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 420 VSA_STATUS: 8 VSA
+ SG_ USER_BRAKE : 7|16@0+ (0.015625,-103) [0|1000] ""  NEO
+ SG_ ESP_DISABLED : 28|1@0+ (1,0) [0|1] ""  NEO
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] ""  NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 423 XXX_5: 2 XXX
+
+BO_ 427 XXX_6: 3 VSA
+
+BO_ 428 XXX_7: 8 XXX
+
+BO_ 432 STANDSTILL: 7 VSA
+ SG_ WHEELS_MOVING : 12|1@0+ (1,0) [0|1] ""  NEO
+ SG_ BRAKE_ERROR_1 : 11|1@0+ (1,0) [0|1] ""  NEO
+ SG_ BRAKE_ERROR_2 : 9|1@0+ (1,0) [0|1] ""  NEO
+ SG_ COUNTER : 53|2@0+ (1,0) [0|3] ""  NEO
+ SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 450 XXX_8: 8 EPB
+ SG_ EPB_ACTIVE : 3|1@0+ (1,0) [0|1] ""  NEO
+ SG_ EPB_STATE : 29|2@0+ (1,0) [0|3] ""  NEO
+
+BO_ 464 WHEEL_SPEEDS: 8 VSA
+ SG_ WHEEL_SPEED_FL : 7|15@0+ (0.002759506,0) [0|70] "m/s"  NEO
+ SG_ WHEEL_SPEED_FR : 8|15@0+ (0.002759506,0) [0|70] "m/s"  NEO
+ SG_ WHEEL_SPEED_RL : 25|15@0+ (0.002759506,0) [0|70] "m/s"  NEO
+ SG_ WHEEL_SPEED_RR : 42|15@0+ (0.002759506,0) [0|70] "m/s"  NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] ""  NEO
+
+BO_ 469 XXX_9: 3 XXX
+
+BO_ 470 XXX_10: 2 VSA
+
+BO_ 474 XXX_11: 8 XXX
+
+BO_ 476 XXX_12: 7 XXX
+
+BO_ 479 XXX_13: 8 XXX
+
+BO_ 493 XXX_14: 5 XXX
+
+BO_ 495 XXX_15: 8 XXX
+
+BO_ 493 XXX_16: 5 VSA
+
+BO_ 495 XXX_17: 8 XXX
```

Comment:
This message (0x1EF) seems to correlate with ACC. 
03 FF 00 75 30 00 00
Byte 1 is 03 when ACC off, changes to 83 when ACC is on. 
Byte 3 occasionally goes up, looks like this happens when ACC speed is lowered and car is traveling faster than set limit.